modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -512,10 +534,14 @@ protected void afterStateChangeFinished(IgniteUuid msgId, boolean success) {
     ) {
         DiscoveryDataClusterState state = globalState;
 
-        if (log.isInfoEnabled())
-            U.log(log, "Received " + prettyStr(msg.activate()) + " request with BaselineTopology" +
-                (msg.baselineTopology() == null ? ": null"
-                    : "[id=" + msg.baselineTopology().id() + "]"));
+        if (log.isInfoEnabled()) {
```

Comment:
Sergey,

There is no need to add if (log.isInfoEnabled()) the U.log implementation already has it.

Reply:
Done!